packages/vat-data/src/vat-data-bindings.js

```diff
@@ -91,19 +93,115 @@ export const partialAssign = (target, source) => {
 // @ts-expect-error TODO statically recognize harden
 harden(partialAssign);
 
+/**
+ * @type {WeakMap<import('./types.js').Baggage, Map<string, (k?:string) => any>>}
+ */
+const collisions = new WeakMap();
```

Comment:
they're not collisions until a collision, right?

please name to something more accurate. e.g. `submapsSeen`

Reply:
Used `provisionTracker`. Done